apps/daimo-mobile/src/view/screen/link/NoteScreen.tsx

```diff
@@ -189,11 +194,8 @@ function NoteDisplayInner({
           <TextBold>Claimed by {getAccountName(noteStatus.claimer!)}</TextBold>
         );
       case "cancelled":
-        if (isOwnSentNote) {
-          return <TextBody>You reclaimed this payment link</TextBody>;
-        } else {
-          return <TextError>Cancelled by sender</TextError>;
-        }
+        assert(isOwnSentNote, "cancelled note not sent by user");
+        return <TextBody>You reclaimed this payment link</TextBody>;
```

Comment:
`You cancelled...`

Reply:
deferring to #571 -- will do it in a separate PR since there's a lot of these